src/core/Akka.API.Tests/verify/CoreAPISpec.ApprovePersistence.DotNet.verified.txt

```diff
@@ -1056,16 +1056,18 @@ namespace Akka.Persistence.Journal
     public class MemoryJournal : Akka.Persistence.Journal.AsyncWriteJournal
     {
         public MemoryJournal() { }
-        protected virtual System.Collections.Concurrent.ConcurrentDictionary<string, System.Collections.Generic.LinkedList<Akka.Persistence.IPersistentRepresentation>> Messages { get; }
```

Comment:
So I struggled with these API changes here, but ultimately concluded that they are necessary because:

1. We do need to improve / fix the security
2. The `SharedMemoryJournal` has to inherit from the `MemoryJournal`